Products/CMFPlone/tests/testInterfaces.py

```diff
@@ -297,5 +297,5 @@ class KlassInterfaceTest(zope_interface_test):
 def test_suite():
     suite = unittest.TestSuite()
     for test in tests:
-        suite.addTest(unittest.makeSuite(test))
+        suite.addTest(unittest.defaultTestLoader.loadTestsFromTestCase(test))
     return suite
```

Comment:
The Zope testrunner has supported discovery of test classes without a test_suite function since 2009.

Reply:
Never mind, I see this is related to some test classes that are generated above, so maybe it is still needed.